oam/install_scripts/mariadb-columnstore-start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This script allows to gracefully start MCS
+
+/bin/systemctl start mcs-loadbrm
+/bin/systemctl start mcs-workernode
+/bin/systemctl start mcs-controllernode
+/bin/systemctl start mcs-exemgr
```

Comment:
This is not the same ordering that systemd unit ordering contains. Should we change that ordering to match this or vice-versa?

Reply:
Despite the fact we can rely on systemd dependenices here I'll change the order to make it explicit for an external viewer.